### DISCUSSION TITLE
Problem with verification

### DISCUSSION THREAD

**Discussion by ARTEMENKOSERGEYVL** [RosettaCommons/rosetta#215] [ROSIE]
Hi. I have a problem with verification ''Your GitHub account does not have verified academic email address!''. 
Can you help my with it? 
My academic email address: s.v.artemenko@utmn.ru

**Comment by roccomoretti** [comment]
Unfortunately, due to the ongoing situation in eastern Europe, there are legal/administrative issues with using servers in the United States to provide computational services to people in Russia, which means that we are unable to approve Russian email addresses for ROSIE accounts at this time.

We apologize for the current state of the world.

**Comment by ElkeDeZitter** [comment]
Hi,
I have the same problem, while not located or affiliated to Russia.
At first my primary e-mail address was not an academic one, hence I changed it. Nevertheless it is still not recognised as an institutional e-mail address.

**Reply by roccomoretti** [reply]
@ElkeDeZitter Please open up a separate discussion topic, and in it be sure to specify the email domain (the bit after the `@`) for your institution. If we can verify it's an academic institution (or rough equivalent), we can add the domain manually.

**Comment by gustko** [comment]
Hi,
Struggeling with the same problem. I'm working at the Norwegian University of Life Sciences for which the email domain is @nmbu.no. Would it be possible to add this domain manually? Would be greatly appreciated.

cheers,
Gustav

**Reply by roccomoretti** [reply]
I'm not sure this message was seen. If you still are having issues, please open up a separate discussion topic.
